docs/user/filenodes.md

```diff
@@ -0,0 +1,56 @@
+# FlowForge File Nodes
+
+Projects running within FlowForge include a set of nodes that make it possible
+to store files safely regardless of the environment. For edge devices the files 
+are stored locally, containerized applications can read and write to persistent 
+storage via API calls to a file-server instance by using these nodes.
+
+*NOTE: This set of nodes allow you to read and write persistent files in a containerised
+environment only when coupled with a [file-storage](../install/file-storage/README.md) instance*
+
+
+## Nodes
+
+There are two nodes in the File Node collection:
+
+- `file` - A file node for writing to persistent storage
+- `file in` - A file node for reading from persistent storage
+
+## Install
+
+### FlowForge
+
+The File Nodes are installed automatically with FlowForge however in order for a 
+project to use them, the built-in Node-RED file nodes must first be disabled. 
+This is done by simply adding `10-file-js` in the **Exclude nodes by filename** 
```

Comment:
This is not true, I cannot edit this.

<img width="463" alt="Screenshot 2022-12-20 at 09 19 54" src="https://user-images.githubusercontent.com/2529595/208617817-99ef70d8-4cc7-464d-9a4e-76731ddbab76.png">

Reply:
It is true however your particular account is not permitted to edit this.

How should this be worded?

#### it could be simplified to...
" The File Nodes are installed automatically with FlowForge however they must be enabled by a project owner so long as the platform administrator has permitted this in the project template"

####  or it could be more technical like...

" The File Nodes are installed automatically with FlowForge however in order for a 
project to use them, the built-in Node-RED file nodes must first be disabled. 
This is done by simply adding `10-file-js` in the **Exclude nodes by filename** 
section of your FlowForge projects settings under  the **Palette** section.
**NOTE:**  The **Exclude nodes by filename** setting can only be modified by a project owner and the project template must permit an owner to edit this field. The project template can only be edited by an administrator of the platform. "

Please advise.

Reply:
revised in new commits